java/restheart/pom.xml

```diff
@@ -18,6 +18,21 @@
         <artifactId>restheart</artifactId>
         <version>${restheart.version}</version>
       </dependency>
+
+      <!-- avoids including following artifacts in shaded jar-->
+      <dependency>
+        <groupId>org.graalvm.polyglot</groupId>
```

Comment:
Not sure why graalvm here ?

The Dockerfile specify temurin.

PS : I plan to make all java framework run on temurin and graalvm to check also memory usage

Reply:
setting those transitive dependencies of restheart will avoid including them in the shaded jar. Why this is happening is not yet clear, but we are fixing it in our pom.xml. latest version still requires this (however it would just produce some warnings by the maven shade plugin)

btw those dependencies are used by restheart to execute JavaScript plugins (when running on GraalVM). See https://restheart.org/docs/plugins/core-plugins-js 

For this release, better to exclude them defining their scope as `provided`

Reply:
Executing the tests on both Temurin and GraalVM would be absolutely fantastic 🚀! I would love to see this implemented!!

Reply:
for this transitive dependencies, we have now an issue. hope to fix it soon https://github.com/SoftInstigate/restheart/issues/510

Reply:
btw I found a better way to exclude transitive deps...